deployment/jobs/integration-test-job.yaml

```diff
@@ -15,11 +15,11 @@ spec:
       restartPolicy: Never
       containers:
       - name: entitytester
-        image: quay.io/n1analytics/entity-app
+        image: data61/anonlink-app:v1.12.0
         imagePullPolicy: Always
         env:
           - name: ENTITY_SERVICE_URL
-            value: https://beta.anonlink.data61.xyz/api/v1
+            value: https://anonlink.easd.data61.xyz/api/v1
```

Comment:
why do we define the server differently to the benchmark yaml? Is there a reason for this inconsistency?
Wouldn't it be nicer if they both accepted the same name/value pair?

Reply:
Sure would be -> #475